package.json

```
@@ -143,7 +143,7 @@
   "bundlesize": [
     {
       "path": "./dist/instantsearch.production.min.js",
-      "maxSize": "66 kB"
+      "maxSize": "66.15 kB"
```

```suggestion
      "maxSize": "66 kB"
```

If we go for only the specified plugins